common/Fabric-Install-Utilities.psm1

```diff
@@ -318,8 +315,11 @@ function Add-ApiRegistration($authUrl, $body, $accessToken)
             Write-Success "API Resource $($apiResourceObject.name) is already registered...updating registration settings."
             Write-Host ""
             try{
-                $registrationResponse = Invoke-RestMethod -Method Put -Uri "$url/$($apiResourceObject.name)" -Body $body -ContentType "application/json" -Headers $headers
-                return ""
+                Invoke-RestMethod -Method Put -Uri "$url/$($apiResourceObject.name)" -Body $body -ContentType "application/json" -Headers $headers
+                
+                # Reset api secret
+                $apiResponse = Invoke-RestMethod -Method Post -Uri "$url/$($apiResourceObject.name)/resetPassword" -ContentType "application/json" -Headers $headers
```

Comment:
I believe we will need to make a similar change to the Add-ClientRegistration function.